test/cypress/integration/card/email-typo.spec.js

```diff
@@ -0,0 +1,321 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+const lodash = require('lodash')
+
+describe('Standard card payment flow', () => {
+  const tokenId = 'be88a908-3b99-4254-9807-c855d53f6b2b'
+  const chargeId = 'ub8de8r5mh4pb49rgm1ismaqfv'
+
+  const invalidEmail = 'validpayingemail@gnail.com'
```

Comment:
Not really sure about the term `invalidEmail` because it might be valid. How about `likelyTypoEmail` or something?